### DISCUSSION TITLE
1

### DISCUSSION THREAD

**Discussion by ghost** [Pithus/bazaar#92] [Q&A] [answered]

**Comment by evilcel3ri** [comment, marked as answer]
Hello! Thank you for reaching out. Pithus is an open source project, we'd be delighted if you use it in your class. As Pithus is still in Beta, please let us know if you or your students find errors or bugs, or, more generally, would like to suggest improvements and new features :)

**Reply by evilcel3ri** [reply]
I think you're already on our discord :) If we need help for anything, we'll let people know over there. Have a nice day!
